src/assets/colors.tsx

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable prettier/prettier */
+const COLORS = {
+  black: '#000000',
+  white: '#ffffff',
+  primary: '#2B50EC',
+  secondary: '#21CEA1',
+};
```

Comment:
Vergeet hun donkerblauw ook niet mee te nemen hierin: #111e55 (dit heb ik namelijk als primary in het dashboard en het contrast is daardoor veel beter)

Reply:
Goed als ik de lichtere blauw naar tertiary verander dan?

Reply:
Check